addons/scratch-notifier/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Scratch Notifier",
-  "description": "Notifications on new Scratch messages. Click the notification to go straight to the message (comment, project, etc.)",
+  "description": "Receive notifications when you get new Scratch messages. Click the notification to go straight to the message (comment, project, etc.)",
```

Comment:
You removed imperative from a lot of other addon descriptions - is it really appropriate here?